README.md

```diff
@@ -312,6 +312,42 @@ module.exports = BaseApp.extend({
 
 ```
 
+### Template Engines
+
+While Template Adapters provide the layer of abstraction that allow you to use your favorite template engine in a Rendr app, the Template Engine option itself will tell the app which version to use exactly. 
+The default is set to be Handlebars, which is currently supported by the Rendr-handlebars adapter until version 2.0.0.
+**When setting up your Rendr app, you'll need to add your Template Engine of choice to package.json.**
+ 
+E.g.
+ 
+```js
+// /package.json
+
+"dependencies": {
+  ...
+  "express": "^4.12.0",
+  "handlebars": "^2.0.0"
+  "qs2": "~0.6.6",
+  ...
+},
+  
+```
+
+####Using Custom Template Engines
+
+You can tell Rendr which Template Engine to use.  This represents the node-module that contains the engine.
+
+```js
+// /app/app.js
+
+module.exports = BaseApp.extend({
+  defaults: {
+    templateEngine: 'jade'
```

Comment:
The rest looks good, i just thought you might have unlocked some secret about how to get jade to work, and i was so excited. ahaha.

Can you also add this documentation to the documentation site? https://github.com/rendrjs/rendrjs.github.io You should be able to just modify it on github w/ the markdown editor.